### PR TITLE
Mwpfe 35 contacts form component

### PR DESCRIPTION
**Pull request by Martynas-J**
I have added a contact form that sends data to gmail

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -13,5 +13,8 @@
   "prettier.singleQuote": true,
   "editor.formatOnType": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "[json]": {
```
Comment (mbitc) [inline .vscode/settings.json:17]: why not Prettier?

File: src/components/ui/FormItem.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          className='text-red-500 text-xs italic'
+        />
+      </div>
+      <div className='mb-4'>
+        <button
+          type='submit'
+          className='bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline'
+        >
+          {formInputsTexts[4].inputText}
+        </button>
```
Comment (mbitc) [inline src/components/ui/FormItem.jsx:97]: We have button component...

**Review by mbitc — CHANGES_REQUESTED** [review]
Easy job, few place

File: src/components/ui/FormItem.jsx (hunk 2 above)
Comment (Martynas-J) [reply src/components/ui/FormItem.jsx]: I fixed it

**Review by mbitc — APPROVED** [review]

**Pull request merged**